README.md

```diff
@@ -22,5 +22,15 @@
 - Exactly same like any other plugins. Users need to upload the files inside `upload` folder of the package maintaining the already made structure using any FTP / WebDav and install / activate the plugin from ACP.
 - Users need to create App for the website at the service provider's developer site to obtain App ID / Secret which are required to input through MyBB ACP Plugin settings. For detail guidelines on creating App for individual service providers please refer [Wiki](https://github.com/mybbgroup/isango/wiki).
 
+The url for all gateways is:
```

Comment:
Please change it to `The callback url for all gateways is:`

Reply:
Done!